python/sglang/srt/layers/moe/topk.py

```diff
@@ -325,7 +325,7 @@ def biased_grouped_topk_gpu(
         and is_power_of_two(correction_bias.shape[0])
     ):
         topk_weights, topk_ids = moe_fused_gate(
-            gating_output,
+            gating_output.to(dtype=torch.float32),
```

Comment:
my main concern is whether this will slow down bs1. if e.g. @zhyncs is ok about that then LGTM

Reply:
yep